src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +11,14 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        UserService userService = new UserService();
```

Comment:
make this service a private class field to avoid redundant recreation per each method invocation

Reply:
Thanks, fixed